### PR TITLE
*: always enable block cache

### PR DESCRIPTION
**Pull request by BusyJay**
### What is changed, added or deleted? (Required)

TiKV configuration storage.block-cache.enabled is removed.

Check https://github.com/tikv/tikv/pull/13903.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v6.6 (TiDB 6.6 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tikv-configuration-file.md [hunk 1]
```diff
@@ -1145,7 +1140,7 @@ rocksdb defaultcf、rocksdb writecf 和 rocksdb lockcf 相关的配置项。
 
 ### `block-cache-size`
 
-+ 一个 RocksDB block 的默认缓存大小。
++ 一个 RocksDB block 的默认缓存大小。该配置从 6.6 以后仅用于计算 storage.block-cache.capacity 的默认值。
```
Comment (Oreoxmt) [inline tikv-configuration-file.md:1143]:
```suggestion
+ 一个 RocksDB block 的默认缓存大小。从 v6.6.0 起，该配置仅用于计算 `storage.block-cache.capacity` 的默认值。
```

File: tune-tikv-memory-performance.md [hunk 2]
```diff
@@ -72,6 +72,7 @@ log-level = "info"
 ## 在大多数情况下，可以通过 LRU 算法在各 CF 间自动平衡缓存用量。
 ##
 ## `storage.block-cache` 会话中的其余配置仅在开启 `shared block cache` 时起作用。
+## 从 6.6 开始，该选项永远开启且无法关闭。
```
Comment (Oreoxmt) [inline tune-tikv-memory-performance.md:75]:
```suggestion
## 从 v6.6.0 开始，该选项永远开启且无法关闭。
```

**Review by Oreoxmt — APPROVED** [review]
Rest LGTM

**Review by tabokie — APPROVED** [review]

**Review by shichun-0415 — APPROVED** [review]
LGTM

**Pull request merged**